PROTO_tests/tests/io_test.py

```diff
@@ -16,6 +16,7 @@
 
 NUMERIC_TYPES = ["int64", "float64", "bool", "uint64"]
 NUMERIC_AND_STR_TYPES = NUMERIC_TYPES + ["str"]
+COMPRESSIONS = ["snappy", "gzip", "brotli", "zstd", "lz4"]
```

Comment:
We already have `COMPRESSIONS` defined later so I think this might get overwritten

Reply:
Fixed.